### PR TITLE
helm: Set --enable-hubble/--hubble-socket-path flags

### LINKED ISSUE #10740: update helm template to control --enable-hubble/--hubble-socket-path flags
right now you cannot enable hubble from helm.

ref: https://github.com/cilium/cilium/commit/f6e82ec8653288397d9c7af2f0565895ee7a95f4

### PR DESCRIPTION
**Pull request by michi-covalent**
Fixes: #10740, #10826, #10827

Signed-off-by: Michi Mutsuzaki <michi@isovalent.com>

### CODE REVIEW

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
test-me-please

**Review by rolinh — APPROVED** [review]
LGTM, thanks for having taken care of this!

**Comment by michi-covalent** [comment]
test-me-please

**Comment by gandro** [comment]
test-me-please

File: install/kubernetes/cilium/charts/config/templates/configmap.yaml [hunk 1]
```diff
@@ -360,10 +360,11 @@ data:
 {{- else }}
   operator-api-serve-addr: '[::1]:9234'
 {{- end }}
-{{ if gt (len .Values.global.hubble.listenAddresses) 0 }}
-  # A space separated list of addresses for Hubble server to listen to. Currently only unix
-  # domain sockets are supported. Hubble gRPC endpoint will be disabled if this field is not
-  # set.
+{{ if .Values.global.hubble.enabled }}
```
Comment (errordeveloper) [inline install/kubernetes/cilium/charts/config/templates/configmap.yaml:364]: where is this block being closed?

Comment (aanm) [reply install/kubernetes/cilium/charts/config/templates/configmap.yaml:364]: line 391.

**Review by aanm — APPROVED** [review]

File: install/kubernetes/cilium/charts/config/templates/configmap.yaml (hunk 1 above)
Comment (aanm) [reply install/kubernetes/cilium/charts/config/templates/configmap.yaml:364]: I've opened #10826 and #10827 because of this

**Comment by gandro** [comment]
test-with-kernel

File: install/kubernetes/cilium/charts/config/templates/configmap.yaml (hunk 1 above)
Comment (michi-covalent) [reply install/kubernetes/cilium/charts/config/templates/configmap.yaml:364]: fixed

**Comment by gandro** [comment]
test-me-please

**Pull request merged**